aztk/node_scripts/setup_host.sh

```diff
@@ -30,7 +32,13 @@ if ! host $HOSTNAME ; then
     echo $(hostname -I | awk '{print $1}') $HOSTNAME >> /etc/hosts
 fi
 
-if [ $gpu_enabled == "True" ]; then
+# Install docker-compose
```

Comment:
this isn't actually used in this PR. Can we comment it out/remove it until it is since this will add to cluster startup time?

Reply:
Pablo is using it in his plugin. This allow plugins right now to use docker compose